hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/datanode/BlockReceiver.java

```diff
@@ -598,7 +599,9 @@ private int receivePacket() throws IOException {
       return 0;
     }
 
-    datanode.metrics.incrPacketsReceived();
```

Comment:
I see you've updated it to increment packetsReceived only when seqno is not HEART_BEAT_SEQNO. This is a critical change for accurately tracking packets. However, it would be helpful to add a comment here explaining why heartbeats are excluded from this metric for clarity and future reference.

Reply:
@ashutoshcipher Sir, thanks for your advise. The documentation of packetsReceived metric is showed as below:
<img width="1214" alt="image" src="https://github.com/apache/hadoop/assets/25115709/30c8797f-ce98-40d3-8c92-968b7b2c4ada">

Its original goal is counting total packets received by DataNode excluding heartbeat packet from client.